src/lightning_app/core/app.py

```diff
@@ -162,6 +165,12 @@ def __init__(
         # this should happen once for all apps before the ui server starts running.
         frontend.update_index_file(FRONTEND_DIR, info=info, root_path=root_path)
 
+        if debugger_is_active() and not bool(int(os.getenv("LIGHTNING_DISPATCHED", "0"))):
+            os.environ["LIGHTNING_DISPATCHED"] = "1"
+            from lightning_app.runners import MultiProcessRuntime
+
+            MultiProcessRuntime(self).dispatch()
```

Comment:
should we control whether we are running on the cloud and emit an error in that case? probably it won’t happen since gettrace will be very likely to return None in that case, but better safe than sorry

Reply:
Done !